app/models/user.js

```diff
@@ -0,0 +1,11 @@
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define('users', {
```

Comment:
Sequelize convention states that the table name goes in singular (and not plural) here. So 'user'. It's wierd.